src/gen/conditions.rs

```diff
@@ -314,7 +313,7 @@ fn parse_args(
     }
 }
 
-#[derive(Serialize, Deserialize, Debug)]
```

Comment:
These types were marked `Serialize` and `Deserialize` by mistake (as far as I can tell). It was exposed when changing this to `Streamable` because `NodePtr` does not implement the `Streamable` trait. If it worked previously, it would not have produced any useful results (since `NodePtr` is an `int` under the hood).